test/es6ToFHIRTest.js

```diff
@@ -44,4 +44,26 @@ describe('#ToFHIR', () => {
     });
   });
 
+  describe('#BloodPressureEntry()', () => {
+    const BloodPressureEntry = importResult('shr/slicing/BloodPressureEntry');
+    it('should serialize to a validated BloodPressureEntry instance', () => {
+      const json = context.getJSON('BloodPressureEntry', false);
+      const entry = BloodPressureEntry.fromJSON(json);
+      const fhir = entry.toFHIR();
+      expect(fhir).is.a('object');
+      context.validateFHIR('BloodPressureEntry', fhir);
```

Comment:
I updated the following test, but I think this one actually is doing a decent job (based on what's in the JSON schema) of testing the instance data.

Reply:
I just `console.log`ged the output from this test and saw that the actual is:
```
{
  "resourceType": "Observation",
  "component": [
    "Systolic",
    "Diastolic"
  ]
}
```

There are a few issues with this:
1. It's not valid FHIR.  `Observation.component` is an array of `BackboneElement`s -- which are semi-anonymous inner structured types.  Here's the differential from the `BP` profile this instance is supposed to conform to (the snapshot was too bulky to paste here):
![image](https://user-images.githubusercontent.com/2278253/45047463-18339380-b047-11e8-9543-c1deff4f3a36.png)
The instance that `toFHIR` is producing is an array of `string`s, not an array of objects as would be required.
2. I see that in the CIMPL, `SystolicPressure` and `DiastolicPressure` are defined to have `string` values, so that partially explains it, except in the mapping, it still maps the value to `component.code`:
![image](https://user-images.githubusercontent.com/2278253/45047595-782a3a00-b047-11e8-97b8-483889bec564.png)
Based on this mapping, I'd still at least expect to see `component` represented as an array of objects with `code` properties.  So it seems that the issue is not just how it's represented/mapped in CIMPL but in the `toFHIR` generation itself.
3. That said, a more accurate representation in CIMPL would be for the `SystolicPressure` and `DiastolicPressure` elements to have `Quantity` values (mapped to `component.valueQuantity`) and `Type` properties (mapped to `component.code`). In fact, I think [these Systolic and Diastolic definitions](https://github.com/standardhealth/shr_spec/blob/e437464eba600c759e835cf63a010c12627c7e08/shr_vital.txt#L112-L122) and [this mostly commented out mapping](https://github.com/standardhealth/shr_spec/blob/e437464eba600c759e835cf63a010c12627c7e08/shr_vital_map.txt#L36-L42) from shr_spec represent a correct representation of what you're trying to do here.

And to bring it back around -- if the fixtures had been the actual expected FHIR JSON instances (instead of JSON schema), I would have noticed this issue right away instead of on the 2nd review.